client/src/Organisms/BurgerMenu/BurgerMenuStyled.js

```diff
@@ -25,9 +25,10 @@ const BurgerMenuStyled = styled.div`
     grid-area: 3 / 1 / 4 / 2;
```

Comment:
If display is none, you dont need any other css, it wont be there anywya, they can be removed

Reply:
Fixed.